cmd/server/main.go

```diff
@@ -108,29 +152,36 @@ func main() {
 		default:
 			return fmt.Errorf("unknown log-format %q", c.GlobalString("log-format"))
 		}
+
 		return nil
 	}
 
 	app.Action = func(c *cli.Context) error {
-		socketPath := c.String("socket")
+		config := c.App.Metadata["config"].(*server.Config)
+
+		if !config.SELinux {
+			selinux.SetDisabled()
+		}
+
+		if _, err := os.Stat(config.Runtime); os.IsNotExist(err) {
```

Comment:
I know we added this in another PR but I don't like this, either we have a stat for each binary (runc|pause|conmon) or just runc isn't enough.